ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -795,9 +796,10 @@ def ocs_install_verification(
             verify_device_class_in_osd_tree(ct_pod, device_class)
 
     # RDR with globalnet submariner
-    if (
-        config.ENV_DATA.get("enable_globalnet", True)
-        and config.MULTICLUSTER.get("multicluster_mode") == "regional-dr"
+    if config.MULTICLUSTER[
```

Comment:
You still should have used here `get`: `config.ENV_DATA.get("enable_globalnet", True)`

As in the code under deployment.py it's part of another condition :
`if config.multicluster:`

Which is not a case here.